make/targets_list.mk

```diff
@@ -40,7 +40,59 @@ UNSUPPORTED_TARGETS := \
     CC3D_OPBL \
     CJMCU \
     MICROSCISKY \
-    NAZE
+    NAZE \
+    AIORACERF3 \
+    AIR32 \
+    AIRHEROF3 \
+    ALIENFLIGHTF3 \
+    BEEBRAIN_V2D \
+    BEEBRAIN_V2F \
+    BEESTORM \
+    BETAFLIGHTF3 \
+    CHEBUZZF3 \
+    COLIBRI_RACE \
+    CRAZYBEEF3DX \
+    CRAZYBEEF3FR \
+    CRAZYBEEF3FS \
+    DOGE EACHIF3 \
+    FF_ACROWHOOPSP \
+    FF_KOMBINI \
+    FF_PIKOBLX \
+    FF_RADIANCE \
+    FLIP32F3OSD \
+    FRSKYF3 \
+    FURYF3 \
+    FURYF3OSD \
+    IMPULSERCF3 \
+    IRCFUSIONF3 \
+    IRCSYNERGYF3 \
+    ISHAPEDF3 \
+    KISSCC \
+    KISSFC \
+    LUMBAF3 \
+    LUXV2_RACE \
+    LUX_RACE \
+    MIDELICF3 \
+    MOTOLAB \
+    MULTIFLITEPICO \
+    OMNIBUS \
+    RACEBASE \
+    RCEXPLORERF3 \
+    RG_SSD_F3 \
+	RMDO \
```

Comment:
That looks like a TAB instead of 4 spaces.

Reply:
Ech, nothing is right for you today, @docteh. :stuck_out_tongue_closed_eyes: